plugin-server/src/capabilities.ts

```diff
@@ -18,10 +18,21 @@ export function getPluginServerCapabilities(config: PluginsServerConfig): Plugin
             return { ingestion: true, ...sharedCapabilities }
         case 'async':
             return {
+                processPluginJobs: true,
+                processAsyncHandlers: true,
                 pluginScheduledTasks: true,
+                ...sharedCapabilities,
+            }
+        case 'async-no-scheduler':
+            return {
```

Comment:
I'd actually already split out jobs in this PR i.e. mapping capabilities 1:1 with deployments (except http ofc)